Gruntfile.js

```diff
@@ -1,10 +1,15 @@
+/* eslint-env node */
 "use strict";
 
 module.exports = function (grunt) {
     grunt.config.init({
         lintAll: {
             sources: {
-                js:    ["./src/js/**/*.js", "tests/js/**/*.js", "./*.js", "!./src/lib/**/*.js"]
+                js:    ["./src/js/**/*.js", "tests/js/**/*.js", "./*.js", "!./src/lib/**/*.js"],
+                md:    [ "./*.md", "tests/**/*.md", "!tests/fixtures/md/bad.md"],
```

Comment:
You can remove the excluded "bad" references, which are just skipping intentionally broken content in the `gpii-grunt-lint-all` repository's tests.